### PR TITLE
Feature/click fast card opens modal

### PR DESCRIPTION
**Pull request by mattash**
Checkout #26

### CODE REVIEW

**Comment by andylitalo** [comment]
Right now, clicking anywhere inside the image on the fast card opens the modal, including the "Join Fast" and "Leave Fast" buttons, so when clicking either button, the modal starts to pop up before the user is redirected. I think we'll need javascript to exclude them from the clickable region (see [here](https://stackoverflow.com/questions/70861330/exclude-child-components-from-div-onclick)).

Holding of from merging for now.

**Comment by mattash** [comment]
Good catch. I have a good idea how to fix it.

**Comment by mattash** [comment]
I resolved previous comment in https://github.com/surp-hovhannes/bahk/pull/33/commits/9bb659ce86bdbaf8358c729bb70629e7dff40faf

File: hub/templates/components/fast_card.html [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <div class="col-lg-4 col-md-6 col-sm-12 text-center">
-    <div class="card mb-3" style="background-image: url('{{ fast.image }}'); background-size: cover;">
+    <div class="card mb-3" data-bs-target="#{{ fast.modal_id }}" data-bs-toggle="modal" onclick="if(event.target.tagName !== 'BUTTON' && !event.target.closest('form')) { location.href='#'; }" style="background-image: url('{{ fast.image }}'); background-size: cover; cursor: pointer;">
```
Comment (andylitalo) [inline hub/templates/components/fast_card.html:2]: for some reason the modal still starts to open when I click the Join Fast or Leave Fast buttons, even though you're excluding buttons from the event target tag names...

**Review by andylitalo — CHANGES_REQUESTED** [review]
Looks like it should work but modal still opens upon button clicks in local testing :(

**Comment by andylitalo** [comment]
I'm going to approve since I think giving people the ability to learn about the fast by clicking on the card is more valuable than flawless UX :)

**Review by andylitalo — APPROVED** [review]

**Pull request merged**